README.md

```diff
@@ -2,6 +2,50 @@
 
 This repository contains the code responsible for openstates.org, the website and v2 API (graphql).
 
+## Make changes
+
+Changes should be made in a local checkout of the repo. There is a bit of setup to get a working DB and environment,
+which by default will run in docker.
+
+* Branch off of the `main` branch for simplicity (we don't have a DEV environment, so `develop` is not very useful)
```

Comment:
`develop` is what the github action workflows look at to trigger new builds of the docker containers/etc. We can certainly _change_ that to `main`, but we should actually change the repository settings then.

Reply:
Good catch, I wasn't sure why `develop` existed